manifests/e/EclipseAdoptium/Temurin/16/16.0.2.7/EclipseAdoptium.Temurin.16.locale.en-US.yaml

```diff
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: EclipseAdoptium.Temurin.16
+PackageVersion: 16.0.2.7
+PackageLocale: en-US
+LicenseUrl: https://adoptium.net/about.html
+Moniker: temurin16
+Tags:
+- openjdk
+- java
+- jvm
+- jdk
+- temurin
+PackageUrl: https://adoptium.net
+Publisher: Eclipse Adoptium
```

Comment:
```suggestion
Publisher: Eclipse Foundation
```

Reply:
Vendor name has changed to `Eclipse Adoptium` instead of `Eclipse Foundation` that was one of the premises for this change.